docs/getting_started.md

```diff
@@ -122,6 +135,28 @@ If you don't want to use the shared key, you can change this value to
 
 How to connect to the node via SSH you can find [here](./debugging.md).
 
+### Authentication Config
+
+For some services we will require additional information in order to
+successfully authenticate. This is a simple yaml file that you will need to
+create manually.
+
+Create a file under `ocs-ci/data/` named `auth.yaml`.
+
+#### Quay
+
+To authenticate with quay you will need to have an access token. You can
+generate one yourself by following [the API doc](https://docs.quay.io/api/) or
+you may use the one QE has generated already. Ask people on ocs-qe mailing list
+or chat room if you don't know where to find the access token.
+
+To enable ocs-ci to use this token, add the following to your `auth.yaml`:
```

Comment:
Can you please share the token on some of mojo page like we have for pull secret? And add link here?  You can do in different PR if you would like to merge this as you have approvals.

Reply:
@petr-balogh We have the `auth.yaml` in our ecosystem private files directory as well as the token being documented in our passwords file with the other quay information. I can also put this in a mojo page if it will help users, but I was trying to avoid putting it in a publicly available place as for now it's legitimately _my_ token.